### PR TITLE
New Updates

### PR DESCRIPTION
**Pull request by TSterling76**

### CODE REVIEW

**Comment by TSterling76** [comment]
@aaronlippold @camdenmoors

**Comment by em-c-rod** [comment]
How did you test this on your end? It would be good to add a GitHub workflow to automatically test this profile.

**Comment by TSterling76** [comment]
I test  by running Inspec on each file. I'm not sure how to set up a Github workflow here.

File: controls/SV-221580.rb [hunk 1]
```diff
@@ -44,4 +44,15 @@
   tag legacy: ["SV-57625","V-44791"]
   tag cci: ["CCI-001166"]
   tag nist: ["SC-18 (1)"]
-end
\ No newline at end of file
+  
+  describe.one do
+    describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome\') do
```
Comment (HenryXiaoHX) [inline controls/SV-221580.rb]: The last backslash in the registry_key path, the backslash after Chrome, serves as an escape character and have escaped the single quote after it. Consider removing the backslash or adding an extra backlash to fix the syntax.

File: controls/SV-221586.rb [hunk 2]
```diff
@@ -36,4 +36,9 @@
   tag legacy: ["SV-89845","V-75165"]
   tag cci: ["CCI-000169"]
   tag nist: ["AU-12 a"]
-end
\ No newline at end of file
+  
+  describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome') do
+    it { should have_property '\"AllowDeletingBrowserHistory\"'}
+    its('\"AllowDeletingBrowserHistory\"') { should cmp '\"0\"' }
```
Comment (HenryXiaoHX) [inline controls/SV-221586.rb]: The combination of backslash followed by double quotes,`\"`, is not necessary. It checks for `\"AllowDeletingBrowserHistory\"` as opposed to `AllowDeletingBrowserHistory` as the property. Similarly, it checks for `\"0\"` as opposed to `0` as the value. Consider removing this combination `\"` to fix the syntax.

File: controls/SV-221587.rb [hunk 3]
```diff
@@ -34,4 +34,9 @@
   tag legacy: ["SV-94633","V-79929"]
   tag cci: ["CCI-000169"]
   tag nist: ["AU-12 a"]
-end
\ No newline at end of file
+  
+  describe registry_key('HKEY_LOCAL_MACHINE\Software\Policies\Google\Chrome') do
+    it { should have_property '\"PromptForDownloadLocation\"'}
```
Comment (HenryXiaoHX) [inline controls/SV-221587.rb]: The combination of backslash followed by double quotes,`\"`, is not necessary. It checks for `\"PromptForDownloadLocation\"` as opposed to `PromptForDownloadLocation` as the property. Similarly, it checks for `\"1\"` as opposed to `1` as the value. Consider removing this combination `\"` to fix the syntax.

**Review by HenryXiaoHX — CHANGES_REQUESTED** [review]

**Comment by TSterling76** [comment]
Are you seeing the changes?

**Review by HenryXiaoHX — APPROVED** [review]

**Pull request merged**